src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.h

```diff
@@ -87,6 +87,10 @@ class QMCFixedSampleLinearOptimizeBatched : public QMCLinearOptimizeBatched,
   // perform optimization using a gradient descent algorithm
   bool descent_run();
 
+  // Previous default linear optimizer
```

Comment:
I first got confused what "previous" means. Then I realized that it refers to pre "shift" implementations. So please add a note listing the names of "previous"  optimizer.

Reply:
The change is a good move.

Reply:
"rescale" is also contained. So I don't get "default".

Reply:
It was the code that ran prior to the introduction of the "shift" implementations.  Both "rescale" and "quartic" options are included - but they seem more like sub-options than different algorithm implementations.   I said in the description I didn't know what to collectively call this code :)
After the "shift" implementations it seems like this code is run if MinMethod is not in one of those if-tests, so it feels like this code is the fall-through or "default" case (I realize it's not really because MinMethod is initialized to OneShiftOnly)

Reply:
Maybe the function should be named "previous_linear_method_run"?

Reply:
I'm OK with "previous_linear_methods_run" (I changed to prural) and in the documentation, explain that "previous" means.
Or may be change all the function names from XXX_run to run_XXX?

Reply:
Besides "rescale" and "quartic", it appears the code does something else if neither of these is specified.  It calls "lineoptimization2" instead of "lineoptimization3" (what "quartic" uses).   I think calling it "previous_linear_methods_run" is good enough, and I mention "rescale" and "quartic" in the comment.